### PR TITLE
Last changes

### PR DESCRIPTION
**Pull request by SergiiBal**

### CODE REVIEW

File: src/learning/ostap/interfaces/Dog.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+//package learning.ostap.interfaces;
+//
+//import learning.ostap.objects.Cat;
+//import learning.ostap.objects.CatColor;
+//
```
Comment (OstapGonchar) [inline src/learning/ostap/interfaces/Dog.java]: Delete the class, why commented out?

File: src/headToJava/FullMoons.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package headToJava;
+import static java.lang.System.out;
+//import java.io.*;
+import java.security.spec.PSSParameterSpec;
```
Comment (OstapGonchar) [inline src/headToJava/FullMoons.java]: reformat, checkbox cleanup unused imports

File: src/headToJava/TestExeption.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
+package headToJava;
+
+public class TestExeption {
+    public static void main(String[] args) {
+
+        String test = "yes";
+        // "not yes"
```
Comment (OstapGonchar) [inline src/headToJava/TestExeption.java]: remove comment

File: src/headToJava/TestExeption.java [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+try {
+    System.out.println("Начало try");
+    doRisky(test);
+    System.out.println("Конец try");
+} catch (ScaryException se) {
+    System.out.println("Жуткое исключение");
+} finally {
```
Comment (OstapGonchar) [inline src/headToJava/TestExeption.java]: reformat

File: src/kotlin/HelloWorld.kt [hunk 5]
```diff
@@ -0,0 +1,4 @@
+package kotlin
+
+class HelloWorld {
```
Comment (OstapGonchar) [inline src/kotlin/HelloWorld.kt]: delete, weird file

File: src/learning/ostap/collections/Seasons.java [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+        calendar.put("September", "Autumn");
+        calendar.put("October", "Autumn");
+        calendar.put("November", "Autumn");
+        System.out.println(calendar);
+        Map<String, String> calendar2 = new HashMap<>();
+        for (String i : calendar.keySet()) {
+            calendar2.put(calendar.get(i), i);
+        }
+        System.out.println(calendar2);
+    //    calendar2 = calendar.replaceAll();
```
Comment (OstapGonchar) [inline src/learning/ostap/collections/Seasons.java]: cleanup comments?

File: src/learning/ostap/interfaces/Canine.java [hunk 7]
```diff
@@ -0,0 +1,5 @@
+//package learning.ostap.interfaces;
+//
+//interface Canine {
```
Comment (OstapGonchar) [inline src/learning/ostap/interfaces/Canine.java]: delete file

**Review by OstapGonchar — CHANGES_REQUESTED** [review]
Please fix the comments

**Comment by SergiiBal** [comment]
Appied all comments. Please, approve.

**Review by OstapGonchar — APPROVED** [review]
Approved

**Pull request merged**